server/src/controller/manifest.js

```diff
@@ -52,6 +90,15 @@ class ManifestController {
       .get('attributes')
   }
 
+  /**
+   * Update a manifest: create a new content and update the last_update Date
+   *
+   * @public
+   * @param  {Integer}  id      Id of the manifest
+   * @param  {String}   content Content of the manifest
+   * @return {Object}           Content of the manfiest
```

Comment:
This function return a object with the manifest or just the manifest content?

Reply:
Just the manifest content